### DISCUSSION TITLE
Bootstrap Causing White Space on HTML Website

### DISCUSSION THREAD

**Discussion by flakesbest** [twbs/bootstrap#37623] [Q&A]
Hello, Im trying to setup boostrap for the first time and whenever I add html elements like text, boostrap just causes white lines across the space they take up. All Ive done is simply modify one of the quickstart scripts to import my own stylesheet, and added a test title, and this is what happens: https://imgur.com/nMce2ye

Here is my code below:
Index.html
```
<!doctype html>
<html lang="en">
  <head>
    <!-- Required meta tags -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <!-- Bootstrap CSS -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
    <link rel="stylesheet" href="style.css" type="text/css">
  </head>
  
  <body>
    <h3>Title</h3>
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
  </body>
</html>
```
Style.css:
```
html {
  background-color: #151515;
}

h3 {
font-family: 'Source Sans Pro', sans-serif;
  font-weight: 600;
  font-style: normal;
  font-size: 300%;
  color: blue;
  margin-bottom: 0%;
  text-align: left;
  vertical-align: middle;
  padding-top: 10%;
  padding-left: 2.5%;
}
```

I feel like Ive searched everywhere for a solution with no luck, im not sure if I installed it improperly or what, but I would really appreciate the help.

**Comment by julien-deramond** [comment]
I'm not entirely sure to understand what's the issue. So I'll try an answer here.
If we're talking about the white background rendered before the `<h3>` instead of the one you defined (`#151515`), you can specify your rule diffrently with `html > body` instead of `body` or keep your selector and use `!important` to add more importance to your rule so that's applied.
Your issue is probably caused by the fact that Bootstrap has a `body { background-color: #fff }` rule by default.
